File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/localization/LocalizationPackage.java
package abi43_0_0.expo.modules.localization;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi43_0_0.expo.modules.core.BasePackage;
import abi43_0_0.expo.modules.core.ExportedModule;

public class LocalizationPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new LocalizationModule(context));
  }
}

File: android/versioned-abis/expoview-abi45_0_0/src/main/java/abi45_0_0/expo/modules/localization/LocalizationPackage.kt
package abi45_0_0.expo.modules.localization

import android.content.Context
import abi45_0_0.expo.modules.core.BasePackage
import abi45_0_0.expo.modules.core.ExportedModule

class LocalizationPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> =
    listOf(LocalizationModule(context))
}
